<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPerson</class>
 <widget class="QWidget" name="AddPerson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddPerson</string>
  </property>
  <widget class="QPushButton" name="pushButton_sure">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>90</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_personId">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>编  号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_personId">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>90</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>注册成员</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_personName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>姓  名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_personAge">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>年  龄</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_personSex">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>性  别</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_personWeight">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>体  重</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_personHeight">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>身  高</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_personBand">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>绑  定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_personName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_personAge">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_personWeight">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_personHeight">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_man">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_woman">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>210</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ecg">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>90</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>（Kg）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>130</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>（cm）</string>
   </property>
  </widget>
  <zorder>pushButton_sure</zorder>
  <zorder>label_personId</zorder>
  <zorder>lineEdit_personId</zorder>
  <zorder>pushButton_cancel</zorder>
  <zorder>label_2</zorder>
  <zorder>label_personName</zorder>
  <zorder>label_personAge</zorder>
  <zorder>label_personSex</zorder>
  <zorder>label_personWeight</zorder>
  <zorder>label_personHeight</zorder>
  <zorder>label_personBand</zorder>
  <zorder>lineEdit_personName</zorder>
  <zorder>lineEdit_personAge</zorder>
  <zorder>radioButton_man</zorder>
  <zorder>radioButton_woman</zorder>
  <zorder>comboBox_ecg</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit_personWeight</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_personHeight</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
